plans/errata.fmf

```diff
@@ -0,0 +1,5 @@
+summary: Testing builds added to errata
+discover:
+    how: fmf
+    test:
+      - /scanning/oscap-eval
```

Comment:
Currently, we treat Errata testing as sanity testing `oscap-eval`, `anaconda/ospp`, `ansible/ospp`, (I don't see it in last runt but I'd expect also `oscap/ospp`), `rpmbuild-ctest` and few tests we don't have in contest yet (profile name diff check, no rule removed from benchmark check, container scanning).

It might seem as redundant, but I'd like to stay on the safer side here. Our stabilization build might not be identical to what will be built in downstream and thus few sanity checks seem appropriate.

Reply:
Hmm, so maybe it would make sense to:
1. test as much as possible in stabilization
2. test only a few things in gating CI
3. re-test as much as possible in errata testing (incl. manual?)

?

It shouldn't take that much extra resources to just re-run the automated testing on the final downstream content RPM. Either the full stabilization set (all arches) or just the productization test set (without diffs, x86_64 only).

Reply:
Depends how long it will take to re-test everything on all supported RHELs (usually ~7 versions) and archs in Erratum.
Now it's cheap - you get results fast and it's simple to review. I want to keep it like that. The latter part depends if waivers are in good shape, if so it remains simple. The first part to me is unknown, but I don't want to wait 2 days for results when we switch erratum NEW->QE.
With that, I'd prefer the 2nd option you wrote but slightly adjusted - productization test set with diffs